### DISCUSSION TITLE
LoxiLB wins OSS Project of the Year, 2022

### DISCUSSION THREAD

**Discussion by UltraInstinct14** [loxilb-io/loxilb#199] [Announcements]
We are glad to announce that after a tough evaluation of close to 6 months which included scrutiny on code authenticity, feasibility of idea, licensing compliance, functional testing and future value/impact to the open-source community, loxilb is awarded the [grand prize](https://www.oss.kr/dev_competition_activities/show/9a079bda-ef98-4c60-89b0-a59797c5f6bd) as ***Open-Source Project of the Year - 2022*** by Ministry of Science and ICT, South-Korea (our home-base). 

loxilb team is honored and would like to thank all our users, developers and well-wishers for getting us here. Our team is working super hard to crack the stranglehold of  legacy and super-expensive load-balancers and provide simple to use yet powerful cloud-native load-balancer with great features for the benefit of the community at large.

**Comment by CloudyBytes10** [comment]
Keep going. Congrats to the team !!

**Comment by TrekkieCoder** [comment]
One word -Awesome !! 축하합니다.....

**Comment by prudentcircle** [comment]
축하드립니다!
